Src/Doxyfile

```diff
@@ -1604,6 +1620,16 @@ BINARY_TOC             = NO
 
 TOC_EXPAND             = NO
 
+# The SITEMAP_URL tag is used to specify the full URL of the place where the
+# generated documentation will be placed on the server by the user during the
+# deployment of the documentation. The generated sitemap is called sitemap.xml
+# and placed on the directory specified by HTML_OUTPUT. In case no SITEMAP_URL
+# is specified no sitemap is generated. For information about the sitemap
+# protocol see https://www.sitemaps.org
+# This tag requires that the tag GENERATE_HTML is set to YES.
+
+SITEMAP_URL            =
```

Comment:
We can set this aot w/ cmake configure and have source of truth live in a CMakelists.txt file

Reply:
that's a good idea, and I'm sure at some point we'd want to do this (e.g. for putting the correct version number into the documentation). However - for this specific element, I am not sure what it is for or what difference it would make.